src/index.spec.js

```diff
@@ -131,7 +131,7 @@ describe('Mongo', () => {
 
   describe('findUserById', () => {
     it('should return null for not found user', async () => {
-      const ret = await mongo.findUserById('unknowuser');
+      const ret = await mongo.findUserById('589871d1c9393d445745a57c');
```

Comment:
I think throwing an error to indicate that the id isn't a mongo id would be useful to the end user consuming `Accounts.findUserById`, they might not realize their error if the function call just silently returns null.